aviary/utils/Fortran_to_Aviary.py

```diff
@@ -362,6 +362,7 @@ def update_gasp_options(vehicle_data):
         # if the design range target_range value is 0, set the problem_type to fallout
         if design_range[0] == 0:
             input_values.set_val('problem_type', ['fallout'])
+            problem_type = 'fallout'
```

Comment:
```suggestion
            problem_type = 'fallout'
            input_values.set_val('problem_type', [problem_type ])
```

Reply:
this way the string `'fallout'` only has to be entered/updated once if things change

Reply:
agree. done